embedded/EscapeROOM/Makefile

```diff
@@ -0,0 +1,21 @@
+_1_SUB_PROJ_DIR = med_panel
+_2_SUB_PROJ_DIR = radio_morse_panel
+_1_SUB_PROJ_BUILD_DIR = build
+_2_SUB_PROJ_BUILD_DIR = build
```

Comment:
I had an idea to get few `Makefile` variables automatically by using `inline` script
(this would lead to get sub-projects build directories names without typing them manually):
**in top-level Makefile:**
`_1_SUB_PROJ_BUILD_DIR = $(shell cat $(_1_SUB_PROJ_DIR)/Makefile | grep -m 1"BUILD_DIR" | sed 's/ //g' | awk -F '=' '{print $2}')`
`_2_SUB_PROJ_BUILD_DIR = $(shell cat $(_2_SUB_PROJ_DIR)/Makefile | grep -m 1"BUILD_DIR" | sed 's/ //g' | awk -F '=' '{print $2}')`

but i did not  succeed. For whatever reason script does not get further than `grep` command.
So left manually typed ones.

Approve as is?

Reply:
I would say that it is not worth it. Even copying the hex files to top level build directory is optional. I do this only when I need to deploy/send them together.
Minimal top file Makefile would look like this (yes, even "clean" target is optional):
```
all:
        make -C med_panel/ all
        make -C radio_morse_panel/ all
```

Reply:
Got it. Totally agree with you, that this is not worth it.